Code/EnginePlugins/RecastPlugin/Components/DetourCrowdAgentComponent.h

```diff
@@ -0,0 +1,138 @@
+#pragma once
+
+#include <Core/World/Component.h>
+#include <Core/World/World.h>
+#include <RecastPlugin/Components/AgentSteeringComponent.h>
+#include <RecastPlugin/RecastPluginDLL.h>
+
+class ezDetourCrowdWorldModule;
+class ezDetourCrowdAgentComponent;
+struct ezDetourCrowdAgentParams;
+
+//////////////////////////////////////////////////////////////////////////
+
+struct ezDetourCrowdAgentRotationMode
+{
+  using StorageType = ezUInt8;
+
+  enum Enum
+  {
+    LookAtNextPathCorner,
+    MatchVelocityDirection,
+    
+    Default = LookAtNextPathCorner
+  };
+};
+
+EZ_DECLARE_REFLECTABLE_TYPE(EZ_NO_LINKAGE, ezDetourCrowdAgentRotationMode);
```

Comment:
this should use EZ_RECASTPLUGIN_DLL instead of EZ_NO_LINKAGE

Reply:
that is necessary, if it should show up in UI, if it is just for internal use, the DLL export isn't necessary

Reply:
yep, it should show up in UI, gonna fix it